sky/cli.py

```diff
@@ -3643,6 +3643,12 @@ def spot_launch(
       sky spot launch 'echo hello!'
     """
     env = _merge_env_vars(env_file, env)
+    if use_spot is not None:
+        if not use_spot:
```

Comment:
This seems to hint if a CLI flag is used. No hint is printed for the YAML case (see repro in issue).

Reply:
Thanks for pointing that out! Added. 🫡

Reply:
Oh sry just found one bug... Wait a minute

Reply:
Done. Thanks!